content/docs/overview/some_terms.md

```diff
@@ -36,8 +36,37 @@ If you right away want to deep-dive into the topic, you can have a look at the [
 
 ## What are Artifacts?
 
-- `Sources`: input for creating resources e.g. source code
-- `Resources`: contain the code to “do” something, e.g. OCI Images, binaries, etc.
-- Have a component-local identity (name and version)
-- Have a `Type` (similar to a MIME-Type)
-- Have an exchangeable `Access` (formal description how to retrieve an artifact, e.g. a download URL)
+- Can be either of kind
+  - `sources`: input for creating resources e.g. source code or
+  - `resources`: contain the code to “do” something, e.g. OCI images, binaries, etc.
+- Have a component-local `Artifact Identity` (identity attribute set consists of `name`, `extraIdentity` (optional) and `version` (optional))
+- Have a `type` (similar to a MIME-Type, current list is [here](https://github.com/open-component-model/ocm-spec/blob/main/doc/04-extensions/01-artifact-types/README.md))
+- Have an exchangeable `access` (formal description how to retrieve an artifact, e.g. a download URL)
+
+## What are OCM Coordinates?
+
+OCM Coordinates are used to reference OCM Component Versions and artifacts within OCM Component Versions. Coordinates referring to an OCM Component Version are also called `Component Identity`, whereas relative coordinates referring to an artifact are called `Artifact Identity`. `Component Identities` are globally unique and may be used to refer to full Component Versions. `Artifact Identities` are always relative to an OCM Component Version and may only be used in conjunction with a Component Identity.
+
+In detail:
+
+### Component Identity
+
+- Component Name: Identifies a component. Should follow conventions like DNS (start with URL-prefix controlled by the owner of the component)
+- Component Version: If used with a Component name, identifies a specific Component Version.  Must adhere to "relaxed Semver" (major, minor (+optional patchlevel) - optional v-prefix).
+
+### Artifact Identity
+
+Within a Component Version, all artifacts *must* have a unique identity. The identity always includes:
+
+- Kind: Artifacts are either `sources` or `resources`
```

Comment:
as discussed w/ @mandelsoft , we should rather talk of Source Identity and Resource Identity (which both _are_ Artefact Identities) and thus remove `kind`. In other words: in simplest form, artefact identities only have `name` as only attribute.